_o_faq_posts/07.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: single
+title: "How to get information from url?"
+body: "
+While working with angular, many times we need to get data from URL. Data in url can be passed in two ways, first as a parameter and second as a query string.<br/>
+We can get the value of query parameters using ActivatedRoute object, queryParamMap.<br/><br/>
+<b>Accessing Query String Values From URL</b><br/>
+Test URL: <b>http://test-url.com/user-list?type=normal&list=active</b><br/><br/>
+<code>
+ngOnInit() {<br/>
+&nbsp;&nbsp;&nbsp;&nbsp;this.route.queryParamMap.subscribe(queryParams => {<br/>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;this.type = queryParams.get('type');<br/>
+&nbsp;&nbsp;&nbsp;&nbsp;this.list = queryParams.get('list');<br/>
+&nbsp;&nbsp;})<br/>
+}
+</code><br/>
+<b>Accessing the URL Parameters</b><br/>
+Test URL: <b>http://test-url.com/user-list/normal/active</b><br/><br/>
```

Comment:
I think it is not clear from where _type_ parameter is retrieved, I mean in the example url maybe it would be good that appears _.../user-list/{type}/active_